resources/istio-resources/files/dashboards/performance.json

```diff
@@ -67,7 +67,7 @@
       "bars": false,
       "dashLength": 10,
       "dashes": false,
-      "datasource": "Prometheus",
+      "datasource": "${DS_PROMETHEUS}",
```

Comment:
I would suggest to change the datasource to `Prometheus` like it was before. We have been

Reply:
Good catch, my IDE must have miss that.